.editorconfig

```diff
@@ -0,0 +1,14 @@
+root = true
+
+[*]
+end_of_line = lf
+indent_style = space
+trim_trailing_whitespace = true
+insert_final_newline = true
+charset = utf-8
+
+[*.{txt,build,json,xml,xml.in,in,py}]
```

Comment:
As a python dev, we would like to use 4 spaces here :P

Reply:
That's fair :D